content/departments/engineering/teams/code-intelligence/index.md

```diff
@@ -10,115 +12,161 @@ The Code Intelligence team builds tools and services that provide contextual inf
+- We see Code intelligence as the glue that sticks the product together, providing a platform for features from navigation to precise powered searches, compiler-accurate batch changes and insights. We aim to provide support for all widely used languages and for the ones we don’t, provide a platform for any developer to add and test their own indexers.
+- In the near term we want Code intelligence to provide seamless, out-of-the-box, precise code navigation for languages that cover 90% of the market usage.
+- We want our code navigation to reach IDE feature parity, while offering the option of plugging into developers’ favorite IDEs.
+- In the longer term, we envision building a global knowledge graph that accurately maps the entire code universe.
+- Our complete strategy is available at [Code intelligence strategy](../../../../strategy-goals/strategy/code-intelligence/index.md)
+- See [Code Intelligence history](./history.md) for our journey so far!
 
-2017 - Over time we started to experience some issues with language servers. They can be difficult to deploy, slow at runtime, slow to adopt by members of their respective language communities, and slow to develop.
+### Video walkthrough
 
-2018 - We added a new basic code intelligence that is built on search-based heuristics to allow us to provide quick and good enough i.e. "imprecise" support for the [most popular programming languages](https://sourcegraph.com/extensions?query=category%3A%22Programming+languages%22).
-
-2019 - While looking for more efficient alternatives to language servers we found LSIF (Language Server Index Format) which provided an index-based implementation of precise code navigation (similar to srclib) with the advantage of being able to build on top of the work done in the LSP community (LSIF and LSP are sister protocols).
+<div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/fcaddfd333da487cb526a4fc99ead803" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
 
-2020 and beyond - The Code Intelligence team is responsible for how Sourcegraph understands code and for providing data that powers semantic search, batch changes, and code insights. Our API is the point of discovery for the knowledge graph for all source code.
+## Members
 
-## Direction and strategy
+{{generator:product_team.code_intelligence}}
 
-See [Code intelligence strategy](../../../../strategy-goals/strategy/code-intelligence/index.md)
+## How we work
 
-## Overview
+On Slack, use the `#code-intel` channel, tag `@code-intel-support` for any help or use the `@codeintel` handle to notify the whole team. As detailed below in the streams section we do have several additional internal rooms however we expect almost all questions and conversations with external teams/departments to take place in `#code-intel`.
```

Comment:
Are we still #code-intel even after the new marketing efforts?

Reply:
Correct, internally we still are.